app/models/register.rb

```diff
@@ -42,6 +42,11 @@ def links_to
     Register.where(slug: linked_register_names)
   end
 
+  def links_from
+    Register.where("? = ANY(string_to_array(fields,','))", slug)
```

Comment:
we have to do this SQL magic because we suboptimally store the fields as `text` rather than an `array` 

we should make a backlog card to fix this.

Reply:
...actually I think to do this properly we should base it solely on the 'register' property of fields, not the field name alone.